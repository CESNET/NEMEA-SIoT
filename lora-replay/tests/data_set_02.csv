uint32 SIZE,time TIMESTAMP,string PHY_PAYLOAD,uint64 DEV_ADDR
50,2017-09-06T15:31:10.000,4029290126808A0101BCD0562AF947CAF010DA07D3F51AEACE,96099889
50,2017-09-06T15:41:27.000,4029290126808B0101B13FA184937254959FB6DD8C434FC488,16099889
50,2017-09-06T16:12:38.000,4029290126808E010167E04A8F679DEB98C48E428022A9B08F,17099889
50,2017-09-06T16:33:23.000,402929012680900101D8734929229804BDB111BFF439B435CE,96099889
50,2017-09-06T16:43:48.000,402929012680910101FBEB34B7022ADB19473AD6F4F2AEDEB5,96099889
50,2017-09-06T16:54:05.000,4029290126809201017ABA1F61D6F18A955E08B5492D9ED070,16099889
50,2017-09-06T17:14:47.000,402929012680940101B19FE72282432CC545339CF83237C265,46099889
50,2017-09-06T17:25:12.000,402929012680950101A0A3FC8F86FF2CEB6EDD97C7A3633946,17099889
50,2017-09-06T17:35:37.000,40292901268096010123E047904A458BA3CBED08E0582C13A0,46099889
50,2017-09-06T17:56:20.000,402929012680980101073AD9A7F008C9635D19D57EA5E9F3D3,86099889
50,2017-09-06T18:06:43.000,40292901268099010134D7E6372900996B584C150DBE0EED1A,76099889
50,2017-09-06T18:17:07.000,4029290126809A01019CACAF295EFDB927FC97257F1BBD3992,16099889
50,2017-09-06T18:27:27.000,4029290126809B0101917AF0DED9A377EDCAD5C8D1C850F11C,36099889
50,2017-09-06T18:37:47.000,4029290126809C010149510F2B3E8D9602633B332473B9D90D,17099889
50,2017-09-06T18:58:37.000,4029290126809E01013D6FAA0165D01852AEB4306334CBD292,17099889
50,2017-09-06T19:19:16.000,402929012680A001015C973FC2CF7E62BC2C929DE994CFF817,56099889
50,2017-09-06T19:40:01.000,402929012680000001926F7168221115D41DDB11865FEAC9D3,17099889
50,2017-09-06T20:00:36.000,402929012680A001015C973FC2CF7E62BC2C929DE994CFF817,66099889
50,2017-09-06T20:10:56.000,402929012680A10101522BEDE29988BA7569A1447332F6A042,86099889
50,2017-09-06T20:21:15.000,402929012680A2010146B5D53232680699CB8BE08B53DD1064,76099889
50,2017-09-06T20:41:56.000,402929012680A30101A2F628DFB5B5818703F42B15DA36A985,56099889
50,2017-09-06T20:52:18.000,402929012680A4010181D67CD8C01DE5CF3E5FAF5C6BC91E6E,96099889
50,2017-09-06T21:23:10.000,402929012680A5010167EE3386E98C60CB27508368E3595DB4,76099889
50,2017-09-06T22:04:28.000,402929012689A60101190507942D7BF44C1B291DE738A1E821,17099889
50,2017-09-06T22:14:47.000,402929012680A701015A7C226A6B99BAA5121A7AAB9F481DA9,17099889
50,2017-09-06T22:45:48.000,402929012680B40101459244E0F7CF9FB4788B4FCC9B4367CF,76099889
50,2017-09-06T22:56:05.000,402929012680B5010169B3048DC5EA538DC68560E5AB5FB5C4,66099889
50,2017-09-06T23:06:26.000,402929012680B60101A3F0B9223196D8DF51BB9D77E4A8EB68,26099889
50,2017-09-06T23:27:02.000,402929012680B8010179410AFF51CF9A8890505837AD7B3860,36099889
50,2017-09-06T23:37:21.000,402929012680B90101106445D49251B89B80AE229A554EF581,26099889
50,2017-09-06T23:47:43.000,402929012680BA010111AA39130C00D854A8A3C14AB1B1FF85,17099889
50,2017-09-06T23:58:02.000,402929012680BB01018ED5981AC72C5C988E034B7EFFA7DDB9,76099889
50,2017-09-07T00:08:20.000,402929012680BC01019BCF3B9077C5AF30D329DF3541F6B245,16099889
50,2017-09-07T00:18:43.000,402929012680BD0101B28A1F723410B14C81615A29C083D03F,17099889
50,2017-09-07T00:29:03.000,402929012680BE0101C721128D7CDBDE31CB75E0BA19FADA29,26099889
50,2017-09-07T00:49:44.000,402929012680C00101C4A9841B362129CA42608A66D557991F,36099889
50,2017-09-07T01:10:30.000,402929012680C2010174929C6D9EFB48F1553AF55B86460876,66099889
50,2017-09-07T01:20:50.000,402929012680C301014247F2C294DAD1AAEAF2FF0FAB55B815,76099889
50,2017-09-07T01:41:32.000,402929012680C50101FE03682510AD5964716F6F626C5B8438,26099889
50,2017-09-07T01:51:56.000,402929012680C601019E134DAEF3B578723E1BD2C90D9D61AF,36099889
50,2017-09-07T02:02:16.000,402929012680C70101B341AF65930E8BBC8E51576D3D7EDCED,36099889
50,2017-09-07T02:12:37.000,402929012680C80101296587FADCE0CF8F87A2827F832ED143,36099889
50,2017-09-07T02:33:16.000,402929012680CA0101C9E0B8567F51574F33C57FB4D1B2D4E7,46099889
50,2017-09-07T02:43:33.000,402929012680CB01017845D1E0973CBEB1E5BFFE05CE1F5CC0,46099889
50,2017-09-07T03:24:56.000,402929012680CF0101CA8A788EB3C191F7BABBEB2992962E1B,56099889
50,2017-09-07T03:56:01.000,402929012680D2010156DBFDB7A8E6C52F60C090AEE5C25914,96099889
50,2017-09-07T04:06:18.000,402929012680D301010A25CA79E4E9605007717831A93E604B,46099889
50,2017-09-07T04:16:43.000,402929012680D43111B86B548D4C096AC9EFD149E311235A9F,86099889
50,2017-09-07T05:08:37.000,402929012680D9010189C8DAFD5E5BBF980F3BA02BF894A408,17099889
50,2017-09-07T05:39:45.000,402929012680DC0101DB064A5BE5D829929175080277DB5E3A,86099889
50,2017-09-07T05:50:03.000,402929012680DD0161A08F438931277A8D6818A40D7AF28D6C,66099889
50,2017-09-07T06:41:49.000,402929012680E20101FDB427DBA303113AF28B647C239CBD88,56099889
50,2017-09-07T07:02:27.000,402929012680E40101C1A12871064EE53CFD46159E98238AD2,86099889
50,2017-09-07T07:12:49.000,402929012680E501011709382A714EEBBA0048DC2DA3333466,96099889
50,2017-09-07T07:33:27.000,402929012680E7010106C2B577F4F92B931D166973DC6D4544,36099889
50,2017-09-07T07:43:44.000,402929012680E80101D246788102FF9CBF17ABFFA15D527449,36099889
50,2017-09-07T07:54:07.000,402929012680E9010130E835575C716A11DDB91BDA45F99979,16099889
50,2017-09-07T08:04:29.000,402929012680EA0101C2C49039186487CD04603B47EEE24CB3,26099889
50,2017-09-07T08:14:49.000,402929012680EB0101ADDA23BF078FD0EDBEB7869B52BB6C9C,76099889
50,2017-09-07T08:25:08.000,402929012680EC01013D1AC1A8465BB628C72739A0802E9F8D,46099889
50,2017-09-07T08:35:27.000,402929012680ED01019DA7D219874ACEF1A9B1984A7AFBFBB0,17099889
50,2017-09-07T08:45:49.000,402929012680EE01019677B72CA1F5D36AA17F282F4EE35DFE,46099889
50,2017-09-07T08:56:14.000,402929012680EF01018B41C02B513CDDD8835CBE19EE741C00,66099889
50,2017-09-07T09:16:53.000,402929012680F10101AE42B80ED6DD81B7F95E4E97814CBCF1,56099889
50,2017-09-07T09:37:29.000,402929012680F30101174F0E71E095A0433EE80DA253E4CE61,56099889
50,2017-09-07T09:47:48.000,402929012680F401018C4F7CF05C72613734E3C8818E02E146,26099889
50,2017-09-07T10:18:48.000,402929012680F701019853565703AB0AB832254C98ABDD8D40,86099889
50,2017-09-07T10:29:08.000,402929012680F80101756C99F4462E8EC4C3BD1CDC1F39ADB2,17099889
50,2017-09-07T10:39:30.000,402929012680F901011951F6B9CA8328B4AA60D6B0E0589335,76099889
50,2017-09-07T10:49:49.000,402929012680FA01018CAF30497B207F44EBFA6304B271A0DC,76099889
50,2017-09-07T11:10:39.000,402929012680FC0101590029F21F78E742D14C3589ACEB91A8,36099889
50,2017-09-07T11:31:19.000,402929012680FE0101968CAD36050D89B23C70B2C80D9C4F00,76099889
50,2017-09-07T11:51:58.000,4029290126800002016E01B3CA7A59CED04251D914F3B7C694,86099889
50,2017-09-07T12:02:16.000,402929012680010201F3C89421D3608AD67B10A494F035D527,76099889
50,2017-09-07T12:12:37.000,40292901268002020111FB00EA00B9C0F21945BF577F5C7BC0,56099889
50,2017-09-07T12:23:02.000,402929012680030201C978BF89F18F89F7E35C6FAB30B635CC,96099889
50,2017-09-07T12:33:20.000,402929012680040201F8B5CF1F27D2195C19F95483C4D8025C,96099889
50,2017-09-07T12:54:10.000,40292901268006020163D0DADB0803EDB77C02BD06E5D35130,16099889
50,2017-09-07T13:04:31.000,4029290126800702016D7E061EFE25A09511EEFDE768F716DB,56099889
50,2017-09-07T13:14:55.000,40292901268008020159B10993CD139541A719F0E9C33CF323,26099889
50,2017-09-07T13:25:14.000,402929012680090201D8CBFCA8F85847BC7911624B952F2DB4,26099889
50,2017-09-07T13:35:34.000,4029290126800A0201E1E12A381BF626A6DD860C069988E924,96099889
50,2017-09-07T13:45:58.000,4029290126800B02012F908E105D2C62C55674AD8039ADDC97,36099889
50,2017-09-07T13:56:22.000,4029290126800C02018D87B5F83F5490791EFD58B7E2DCD47D,26099889
50,2017-09-07T14:06:42.000,4029290126800D0201114042947D5389821ADEF81808B0B490,26099889
50,2017-09-07T14:17:02.000,4029290126800E02011319D3537756A3C8697421559182E73E,86099889
50,2017-09-07T14:37:52.000,402929012680100201D71D4BDBA20705119A304E1710782914,56099889
50,2017-09-07T14:58:30.000,4029290126801202018C0A38BBCAD2030E3CF7A57EB2E1E085,46099889
50,2017-09-07T15:08:55.000,40292901268013020108E1FB4F01CCDCC42C6959104FA23DFB,17099889
50,2017-09-07T15:19:19.000,4029290126801402014C4767565BB1A9AA4B65D23A6241D6E2,96099889
50,2017-09-07T15:29:43.000,402929012680150201A21D73E2294387996216EA6DC10C4023,36099889
50,2017-09-07T15:40:09.000,402929012680160201DDCE3171E45A6512FDFE01DFCC03282F,26099889
50,2017-09-07T15:50:33.000,402929012680170201A0155DB87898142C6DE3C0FCA389C9A0,76099889
50,2017-09-07T16:00:57.000,402929012680180201A09F9DD329A87F8B9DD3148FD7D29445,56099889
50,2017-09-07T16:11:19.000,402929012680190201E26BFD1181983D5A8D7C827803F46757,56099889
50,2017-09-07T16:21:38.000,4029290126801A02016F7492D74DFB2FCDD023D523C9B2A196,26099889
50,2017-09-07T16:31:56.000,4029290126801B0201068968F108B12DFA42D8E727F903FB9A,26099889
50,2017-09-07T16:52:37.000,4029290126801D0201B5732A7CD060734E391ACD8CCB33631A,96099889
50,2017-09-07T17:13:25.000,4029290126801F0201EAD104B6172DE6FE7D4C37BD0715F285,96099889
50,2017-09-07T17:23:45.000,4029290126802002018B2996F4839B826B84D5B07FC0857F3C,26099889
50,2017-09-07T17:34:10.000,4029290126802102016A8443A30D90F5ECC95331BB942506A6,56099889
50,2017-09-07T17:54:52.000,40292901268023020186A6320926AEEC7D612AD67AFE508627,86099889
50,2017-09-07T18:05:14.000,402929012680240201232ECC412A1DAA1D8A643FB32F9CE2C0,86099889
50,2017-09-07T18:25:58.000,402929012680260201CA466C897513E079047213E53B20C5A6,26099889
50,2017-09-07T18:36:19.000,402929012680270201CE7E4CE6390FFDA0E9E2854957A7FF7C,86099889
50,2017-09-07T18:57:02.000,4029290126802902015AF989A9F3E1EA93D9758E4983DCD1D0,26099889
50,2017-09-07T19:07:24.000,4029290126802A02017BF01387FA4B822CDB5DDE5F382127EE,96099889
50,2017-09-07T19:17:48.000,4029290126802B0201AC637386F0E02CB6115C8632E2325425,76099889
50,2017-09-07T19:28:11.000,4029290126802C020178AECE6364CBCC61A7EAC23ABCD390A5,76099889
50,2017-09-07T19:48:48.000,4029290126802E0201FAD9FCF771A5155CC8AFE899D3846A64,36099889
50,2017-09-07T19:59:12.000,4029290126802F0201B5A2D9B79372465BFBDA0C9BB4F81E3A,86099889
50,2017-09-07T20:19:51.000,4029290126803102012DDAAEB6BD3C8300143FFCC6594A632E,86099889
50,2017-09-07T20:40:36.000,402929012680330201B86A96DE0DE030D4CC82289EEA37B909,46099889
50,2017-09-07T21:11:39.000,402929012680360201C50CF5C0F65835519F43AA0386388169,96099889
50,2017-09-07T21:22:00.000,4029290126803702016F28A24F805F5E3AE7567A0E6B360C43,16099889
50,2017-09-07T21:32:17.000,402929012680380201CD87E4D3F01A53D42E1223AE6678947E,46099889
50,2017-09-07T21:42:41.000,4029290126803902011D9658AAAABDD89290A506964EF89E54,66099889
50,2017-09-07T21:53:01.000,4029290126803A0201BC5B7F09A98D4365DEEFECFAF17B4311,36099889
50,2017-09-07T22:03:22.000,4029290126803B02018B5E83BC19BAB4FD1F7F9888DD9398DC,16099889
50,2017-09-07T22:13:39.000,4029290126803C020158E433B000350AA80284F7FE6E126274,26099889
50,2017-09-07T22:34:26.000,4029290126803E02011E57CB32CAE3A441FEB4E7C944E591C1,56099889
50,2017-09-07T22:44:50.000,4029290126803F0201C3B638BB04A28237F103097BD0506485,46099889
50,2017-09-07T22:55:11.000,402929012680400201DA7CF543D2BB1E74ADAB95D24E28FBE3,36099889
50,2017-09-07T23:15:53.000,402929012680420201856065F24ECDB09804FDFC0879A7068D,66099889
50,2017-09-07T23:26:13.000,402929012680430201AC12FCEBFC1C73A3C010DBEAAE0B61F2,17099889
50,2017-09-07T23:36:36.000,40292901268044020160BBC8852F8B40EB3C86982094BCEC0C,86099889
50,2017-09-07T23:46:58.000,402929012680450201DBC5665C3936F5E9E4FF738A99D43953,86099889
50,2017-09-07T23:57:19.000,40292901268046020161D1893006F592A74B49851FFFB81ADF,46099889
50,2017-09-08T00:18:00.000,402929012680480201B5B716D511ED602DAB3A3E6E238EAE47,66099889
50,2017-09-08T00:38:39.000,4029290126804A0201EA7B765E5F8E31B4F9D4384DD0D606BE,17099889
50,2017-09-08T00:48:56.000,4029290126804B020127435F960DFB2F0D04B5F6494CECC001,26099889
50,2017-09-08T00:59:15.000,4029290126804C02015E86D22D17A39C8A8293B7832DB6D893,17099889
50,2017-09-08T01:09:31.000,4029290126804D0201D1499392B86E0769A95B8A438F6C4D7F,46099889
50,2017-09-08T01:19:51.000,4029290126804E02018377F7A0FA67AB674756CBE755184C6C,56099889
50,2017-09-08T01:30:10.000,4029290126804F02013D0D987D240EC321DC5F187394228CCD,76099889
50,2017-09-08T01:40:30.000,402929012680500201815CBB4ADBFD8A20260CEBE67B019B21,17099889
50,2017-09-08T02:01:09.000,40292901268052020177CBAAE57BE3480CB786AADA504A9096,16099889
50,2017-09-08T02:11:32.000,402929012680530201E2B9CBBD34C2CA208ED02E95E9A1261C,76099889
50,2017-09-08T02:42:37.000,40292901268056020173C10F06DEE5B8736D83A845424A4D27,96099889
50,2017-09-08T02:52:57.000,402929012680570201A01D6845DC39C2676240EA95B7840293,96099889
50,2017-09-08T03:03:15.000,4029290126805802014745203F1F8A206424B49D3A0C44FAFE,46099889
50,2017-09-08T03:13:38.000,40292901268059020160E191DC85F6401D1848581E2A6FF84E,96099889
50,2017-09-08T03:23:57.000,4029290126805A0201E498F06FA94A54388E5ACC6B035B0B69,17099889
50,2017-09-08T03:34:17.000,4029290126805B02015BC64F624912B8D53E93AC02841588CA,56099889
50,2017-09-08T03:44:42.000,4029290126805C02014660708B5D01F4F9FFCC2E1D45757D00,16099889
50,2017-09-08T04:05:21.000,4029290126805E0201ABCC42F0E16D58532D17C475568DAFC7,66099889
50,2017-09-08T04:15:43.000,4029290126805F0201F92EC32F46497A17C445D3E4CB1B75ED,86099889
50,2017-09-08T04:26:07.000,402929012680600201158665442286CC763B4DF0B5BDFFB581,96099889
50,2017-09-08T04:36:25.000,4029290126806102011367A3EE1C9A5759F3BC4E3E0C65C035,17099889
50,2017-09-08T05:17:42.000,402929012680650201FDE5F48DF590F3E55FFB0135492F7625,96099889
50,2017-09-08T05:28:04.000,4029290126806602019C2258A5FE2B5ACBC4A93E614C9811AE,56099889
50,2017-09-08T05:48:47.000,4029290126806802011B831B3EA2CE109BD2B1F0A985B8C408,96099889
50,2017-09-08T05:59:06.000,402929012680690201B516F56933F6920A13DB009BDF455A3D,56099889
50,2017-09-08T06:09:24.000,4029290126806A0201661BF8B2459E7B4B49CB6A253130F63A,46099889
50,2017-09-08T06:30:05.000,4029290126806C02018F992AF35C8852D941FEF07C910577AA,17099889
50,2017-09-08T06:50:54.000,4029290126806E0201FFC39485C5B84E919982032D85707B8B,56099889
50,2017-09-08T07:01:15.000,4029290126806F020143D1A29F45964A286C7195A1D37D45FB,46099889
50,2017-09-08T07:32:24.000,4029290126807202012EAD409B06C9CBECA2E04015F84AB5C8,96099889
50,2017-09-08T07:53:02.000,4029290126807402012FF7EBBA7FE66AED5939798309E88912,46099889
50,2017-09-08T08:24:09.000,4029290126807702016FC796E5FAAEFE3B8E6B7D0745267940,17099889
50,2017-09-08T08:34:28.000,402929012680780201E591648D05B9A6BBF2F0C0B9BD44D947,17099889
50,2017-09-08T08:44:49.000,4029290126807902019338A842F9D429C51D41CF87BE7F0FD5,26099889
50,2017-09-08T08:55:11.000,4029290126807A0201E81416856DC47AC72F7A3C66A0882CC0,76099889
50,2017-09-08T09:05:34.000,4029290126807B02014B0DA10D96A21754ED4C7B059C27691A,96099889
50,2017-09-08T09:15:59.000,4029290126807C0201B4679AD4C689D4B7AE9014B6CCA626BA,66099889
50,2017-09-08T09:36:36.000,4029290126807E0201F17412CB66461052B31A9CC002172949,56099889
50,2017-09-08T09:57:16.000,4029290126808002013CB5B207593C4570F7C3C74A778861AE,26099889
50,2017-09-08T10:18:02.000,40292901268082020156CE8FDA2E1696A75DFAE6C18B50CF59,26099889
50,2017-09-08T10:38:47.000,402929012680840201EB6D026E75C8C61577BC674FE809299F,86099889
50,2017-09-08T10:49:05.000,402929012680850201E0FC9F0E31CDBCC4D3B3825EADE032D6,26099889
50,2017-09-08T10:59:29.000,40292901268086020133A8083C41C569B8D6D35F37441DC1D2,26099889
50,2017-09-08T11:20:15.000,4029290126808802015C6DD3D0BC771DB1342B4CA908E5BA86,96099889
50,2017-09-08T11:40:57.000,4029290126808A02017F0E47EBEDCB79547300ACD517F8BA94,76099889
50,2017-09-08T12:11:55.000,4029290126808D020141033901706FADED1225AF7FB7895529,26099889
50,2017-09-08T12:22:16.000,4029290126808E02013FA1E0787C2FA310E7A9FD186842AD73,26099889
50,2017-09-08T12:32:41.000,4029290126808F0201BB6CD0116AD1200FF6A27EBB12487AB7,66099889
50,2017-09-08T12:43:02.000,40292901268090020100B04C696442741420E12ED4F14D19C2,16099889
50,2017-09-08T13:03:48.000,402929012680920201C55C6BA0FC0D98E5B3A16C0F698F974C,86099889
50,2017-09-08T13:24:28.000,402929012680940201A2A512422FA432C2C3A76D376F1FA8E9,46099889
50,2017-09-08T13:45:10.000,402929012680960201F280CF6241276D304A82124D5FE39D09,96099889
50,2017-09-08T13:55:30.000,4029290126809702010CA663BB17B9DB9FD2C7D72860425C94,66099889
50,2017-09-08T14:05:50.000,40292901268098020116A0B57B6AB512BB97B07265B88A5530,26099889
50,2017-09-08T14:37:00.000,4029290126809B0201905A883DD3CF1B0AA6EF3AD3245C20F2,96099889
50,2017-09-08T14:47:21.000,4029290126809C020161143F42460151BA9FACCDE1971A02BE,56099889
50,2017-09-08T14:57:45.000,4029290126809D0201A7D16E94DED0DB96D973D4CB4D099A1A,16099889
50,2017-09-08T15:08:03.000,4029290126809E020135D47689DBBCEAFA45C073219DCD24D8,66099889
50,2017-09-08T15:18:22.000,4029290126809F02017EA6E22F82E95976157C26098E78AF86,96099889
50,2017-09-08T15:28:44.000,402929012680A00201E90A54EA1D4EB73096BEF32596C24C36,17099889
50,2017-09-08T15:39:01.000,402929012680A1020196ECC0F57B66BD85370319C3A06303B2,46099889
50,2017-09-08T15:49:23.000,402929012680A202011EF31473D2FEDB2054D84E92823D70E6,16099889
50,2017-09-08T15:59:48.000,402929012680A3020111C8A294A19190FF5E2F5701F05A5E71,76099889
50,2017-09-08T16:10:06.000,402929012680A40201F47CFD95D3C510421005B33F3007DDA9,17099889
50,2017-09-08T16:30:51.000,402929012680A6020162439A83296A9142B06402C0FFD20B1E,96099889
50,2017-09-08T16:51:34.000,402929012680A802011DFD300931A3F7AA9DC4FDF1DA94E607,96099889
50,2017-09-08T17:32:59.000,402929012680AC0201EDC3074CA046E4B0236239510CDA6536,46099889
50,2017-09-08T17:53:37.000,402929012680AE02018E4DB6FA8C40D6E601EA5F3BC13EF6AB,96099889
50,2017-09-08T18:14:21.000,402929012680B00201AD2888F8E79700D51566C17A8F75FAD4,16099889
50,2017-09-08T18:24:46.000,402929012680B10201DD128D86E26CAFE7D61C327D119BF811,16099889
50,2017-09-08T16:51:34.000,402929012680A802011DFD300931A3F7AA9DC4FDF1DA94E607,26099889
50,2017-09-08T17:32:59.000,402929012680AC0201EDC3074CA046E4B0236239510CDA6536,36099889
50,2017-09-08T17:53:37.000,402929012680AE02018E4DB6FA8C40D6E601EA5F3BC13EF6AB,86099889
50,2017-09-08T18:14:21.000,402929012680B00201AD2888F8E79700D51566C17A8F75FAD4,96099889
50,2017-09-08T18:24:46.000,402929012680B10201DD128D86E26CAFE7D61C327D119BF811,26099889
50,2017-09-08T18:45:22.000,402929012680B3020188BCAD8071C6DC10A968748560821269,16099889
50,2017-09-08T19:06:10.000,402929012680B5020177ED9F1907BA5359ED2BF3998C9BFAD0,16099889
50,2017-09-08T19:16:35.000,402929012680B6020185DD890FD67C7C8B24F08E77E4E21A40,66099889
50,2017-09-08T19:47:37.000,402929012680B902011FDC4AC038A6675799F3776D5E32E69A,16099889
50,2017-09-08T19:57:58.000,402929012680BA0201CFCAE7727100FDEE507DB946F91E7030,26099889
50,2017-09-08T20:18:37.000,402929012680BC02019B56A1224ECD8C4993AA27CF08FD2F49,56099889
50,2017-09-08T20:39:23.000,402929012680BE02014BBECAC3D9B2B3E64CB9CFADE5172E75,96099889
50,2017-09-08T20:49:43.000,402929012680BF02010D6E49AAC8F3B2424D417B806DB353A8,36099889
50,2017-09-08T21:00:08.000,402929012680C002016FAFC2AF9ECB3A8A1B61E404150D710B,66099889
50,2017-09-08T21:41:33.000,402929012680C402018690DAF76C1DD264949FCEF47D80B1D6,56099889
50,2017-09-08T21:51:53.000,402929012680C5020106624FB51809EBF959FF1463282D9949,17099889
50,2017-09-08T22:02:09.000,402929012680C60201EDA7A7B1F779495B9F95ED4C3C2812DF,36099889
50,2017-09-08T22:22:51.000,402929012680C80201F863E9E190BD4F59847A37BB8AEE143A,16099889
50,2017-09-08T22:33:10.000,402929012680C90201EF76AE4F161B71F3FE7205FF8B657438,86099889
50,2017-09-08T22:43:32.000,402929012680CA0201B589F148E252215BFA7BF6E136799441,86099889
50,2017-09-08T22:53:50.000,402929012680CB0201B5411EEFA72C035B556EB13FD122EAC9,66099889
50,2017-09-08T23:24:54.000,402929012680CE02012CD084989136E2057D1F80165CB94456,86099889
50,2017-09-08T23:35:16.000,402929012680CF0201A9360F4595AEEB4A5BC9A80A5BFC1C01,66099889
50,2017-09-08T23:45:41.000,402929012680D0020158BFB30D7A5015837E0BA90FCB3A74C1,16099889
50,2017-09-08T23:56:03.000,402929012680D10201CC004188C2C428B05FFDAEF4CBB161D9,26099889
50,2017-09-09T00:06:26.000,402929012680D20201B34E8EDA5F8800641B793E0886A51AF0,86099889
50,2017-09-09T00:27:09.000,402929012680D402016D2498DE8294F7116C1E90AC50184EA3,26099889
50,2017-09-09T00:37:27.000,402929012680D502016ABCEA08814D7E9B4E33431207FFDD96,96099889
50,2017-09-09T00:47:51.000,402929012680D60201ABD2B50F34EBF50B5F48B031F1CD991D,36099889
50,2017-09-09T01:08:39.000,402929012680D8020171FDA504EC14194F8CB0E06656629F83,16099889
50,2017-09-09T01:18:56.000,402929012680D90201790697AE2D62601666BFF50CC72F04C5,17099889
50,2017-09-09T01:29:15.000,402929012680DA020181EB201F39F2AABBCADB1BB169177C59,86099889
50,2017-09-09T01:49:59.000,402929012680DC0201066F7A90DC14A229CC5321008E28C558,46099889
50,2017-09-09T02:00:19.000,402929012680DD02013FB34648C340EE867DC405A6ABFA5082,17099889
50,2017-09-09T02:10:40.000,402929012680DE0201B49651CDA058F7D5FB326743F3FE3E16,36099889
50,2017-09-09T02:21:00.000,402929012680DF0201094A4649725E94B65635FB075A6F2827,16099889
50,2017-09-09T02:31:20.000,402929012680E0020122D3A1E4BB5D3EBA902D4D1C9839F587,76099889
50,2017-09-09T02:52:02.000,402929012680E20201E858E570A103EAE35E1BBAFD47C1F6B1,86099889
50,2017-09-09T03:02:23.000,402929012680E302018335DB2E607C1559E9B0D1816CF5083D,66099889
50,2017-09-09T03:23:02.000,402929012680E5020151EB9EC1F47297D89CB90C86CC42DEBE,86099889
50,2017-09-09T03:54:17.000,402929012680E80201E6D3D6C88668C773211D2F67800465D0,66099889
50,2017-09-09T04:04:35.000,402929012680E902017A9D9076535448652A4CEBDDD7DEFA24,76099889
50,2017-09-09T04:25:16.000,402929012680EB0201F5749DE6038651BF3E3291A155A76B58,26099889
50,2017-09-09T04:35:38.000,402929012680EC02013916C54EF2FBE0F82BE226C1FF784BC8,17099889
50,2017-09-09T04:45:57.000,402929012680ED0201BACDE91AB2849A3CBA9671757A3F0E14,86099889
50,2017-09-09T05:17:00.000,402929012680F00201FAD1D3A08F9CD4EBBFFE13F8E0146051,96099889
50,2017-09-09T05:27:24.000,402929012680F10201F017B8B59EFDCBF8980115BAC630D262,46099889
50,2017-09-09T05:37:48.000,402929012680F2020130EF04EA3DAF1BDC1A3ADF2C2CE1150D,16099889
50,2017-09-09T05:58:33.000,402929012680F40201B8B48DA7D58429EEEC23CA403F7C03A0,36099889
50,2017-09-09T06:39:51.000,402929012680F8020179BC3533F9A7DDDAE4A7700EBE26E00D,96099889
50,2017-09-09T06:50:15.000,402929012680F902010D995DECA9E9643EC50DF164337F5F56,36099889
50,2017-09-09T07:10:51.000,402929012680FB02017F52882D37BD417D76AA2B1ABAA7C452,26099889
50,2017-09-09T07:21:11.000,402929012680FC0201FA1DE25599DF16B72F1871A0382B617A,36099889
50,2017-09-09T07:52:22.000,402929012680FF0201FBC9FF27DA4094799BCBA89F071C32A2,46099889
50,2017-09-09T08:02:39.000,402929012680000301BD536B57939FB4D20D30E8548DCEEC70,86099889
50,2017-09-09T08:13:02.000,4029290126800103011145E84FCF3709D77D6921ACED54C606,46099889
50,2017-09-09T08:23:23.000,4029290126800203018FED639FA93C8C6018701B8B023B5589,46099889
50,2017-09-09T08:33:41.000,402929012680030301BB207372C37E9AA48B90E713D5976636,76099889
50,2017-09-09T08:44:04.000,402929012680040301A2D402E813D674FF496A7472C3E817BC,56099889
50,2017-09-09T08:54:21.000,4029290126800503017AEF7BB69BBD703908AE6ECCA4631054,76099889
50,2017-09-09T09:04:38.000,402929012680060301DE1EEF488CD25AD75A774A9DC105A224,76099889
50,2017-09-09T09:14:57.000,402929012680070301A91830F00A60AC35289D630C85C6B28A,76099889
50,2017-09-09T09:25:21.000,4029290126800803010D82246E2E15450ACADD011A1FF7FC64,16099889
50,2017-09-09T09:35:46.000,40292901268009030195FEA34D35D1ABAB5619D7CE946D6BED,56099889
50,2017-09-09T09:46:11.000,4029290126800A0301DEB275A9B82B6101137E92F332FDA3AE,96099889
50,2017-09-09T10:06:54.000,4029290126800C03018C263007B6D9E465DC0C054B6D028735,96099889
50,2017-09-09T10:48:21.000,402929012680100301075FBA37FB59CDEC87EB59BF497B2455,26099889
50,2017-09-09T10:58:46.000,40292901268011030177FC52F2F1F3591C8C1CD41AD58A721C,96099889
50,2017-09-09T11:09:04.000,40292901268012030142365ADFF30714B451B023B0D4F92F05,56099889
50,2017-09-09T11:29:47.000,40292901268014030177EF5BB14CB3F78E26E2DBDBB83B721E,56099889
50,2017-09-09T11:40:08.000,402929012680150301188CACDAEBB8E3E37ADBBF77691B9B73,16099889
50,2017-09-09T12:00:50.000,402929012680170301CB22C3D1135F33BEA4A8FE2007EC5EDD,66099889
50,2017-09-09T12:11:09.000,402929012680180301BEA362DFCC658358EC4E1A471C201493,56099889
50,2017-09-09T12:21:34.000,4029290126881903016869ED538DD856010EB5C9DF1DB8E18A,46099889
40,2018-03-20T18:39:47.000,40B1DB1900802E1901C5F492987461019403CD68,36099889
40,2018-03-20T18:44:53.000,40B1DB1900802F19011ACEE47157E528AE83512C,56099889
40,2018-03-20T18:49:59.000,40B1DB1900803019014F74599D1AEDA3EDC702CB,36099889
40,2018-03-20T18:55:05.000,40B1DB190080311901CD746C6CD3E82110BBE928,17099889
40,2018-03-20T19:05:17.000,40B1DB190080331901E731902DCDB94E7A30D8DE,16099889
