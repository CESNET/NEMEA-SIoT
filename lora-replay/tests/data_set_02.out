66099889,2017-09-06T20:00:36.000,0,"Replay attack has been detected for the device 66099889","01A0"
